include/common/msgType.h

```diff
@@ -693,6 +695,33 @@ struct DB_RESP_Q_msg {
     struct fteid s11_sgw_c_fteid;
  };
 #define S11_DBRESP_BUF_SIZE sizeof(struct DB_RESP_Q_msg)
+
+struct ID_Q_msg{
+    msg_type_t msg_type;
+    int ue_idx;
+    guti _guti;
+
+};
+#define S10_IDREQ_STAGE5_BUF_SIZE sizeof(struct ID_Q_msg)
+
+struct ID_RESP_Q_msg{
```

Comment:
how this is different from id_resp_Q_msg ? and same thing for req message.

Reply:
ID_RESP_Q_msg is a structure for outgoing identification response message, while id_resp_Q_msg is for incoming identification response message.